example/volume.jl

```diff
@@ -1,5 +1,11 @@
-using GLPlot, GLVisualize, GLAbstraction, Colors, GeometryTypes, FileIO
-using Reactive, GLWindow, NIfTI
+using GLPlot, GLVisualize, GLAbstraction, Colors, GeometryTypes
+using Reactive, GLWindow
+
+if !isdefined(:vol)
+    using TestImages
+    vol = testimage("mri-stack")
+end
+
 window = GLPlot.init()
```

Comment:
There is currently no  such method. shouldn't it be 
`window = createdisplay()`?